.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.idea/
```

Comment:
Maybe add this file to your [global git ignore list](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) instead?

Reply:
Sorry I wrongly commit this change. I have removed it.